src/main/resources/templates/macros/report/docstring.vm

```diff
@@ -3,10 +3,11 @@
 #if ($docString)
   <div class="inner-level">
     <div class="docstring indention">
-      <div data-toggle="collapse" class="collapsable-control" data-target="#msg-$docString.hashCode()">
+      #set($docstringId = $counter.getNext())
+      <div data-toggle="collapse" class="collapsable-control" data-target="#msg-$docstringId">
```

Comment:
That is true. ``nextValue()`` should be executed for particular type such as element, step, embedded...